packages/headless/src/features/case-field/case-field-slice.ts

```diff
@@ -27,8 +27,13 @@ export const caseFieldReducer = createReducer(
         state.status.loading = false;
       })
       .addCase(fetchCaseClassifications.fulfilled, (state, action) => {
+        const defaultField = {value: '', suggestions: []};
         Object.entries(action.payload.response.fields).forEach(
           ([fieldName, content]) => {
+            if (!state.fields[fieldName]) {
```

Comment:
@nathanlb I had to add this check when dispatching the `fetchCaseClassification` action when the case fields are not registered.